README.md

```diff
@@ -45,6 +45,17 @@ yarn eject
 Note: this is a one-way operation. Once you eject, you can’t go back!
 If you aren’t satisfied with the build tool and configuration choices, you can eject at any time. This command will remove the single build dependency from your project.
 
+# e2e tests
+
+The following command opens the cypress console from which you can run the tests.
+
+```
+npx cypress open
```

Comment:
Hello @josenaranjo, thanks for working on this!

It would be nice to have an `npm script` that does this. I can think of something like `test:e2e` or just `e2e`.

Reply:
Done